compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -182,9 +182,6 @@ void buildFlowTransformPassPipeline(OpPassManager &passManager,
         return createFormDispatchWorkgroupsPass(
             clDispatchGenerateWorkloadRegion);
       })
-      // TODO(#15003): SCF support appears to be insufficient for stream work.
-      // Need to debug the cause.
-      .addPass(IREE::Flow::createTopLevelSCFToCFGPass)
```

Comment:
follow-on: I noticed an upstream CFG -> SCF pass exists now - would be neat now that you have this plumbed through to start running that (assuming it works :)

Reply:
Lets check into turning this one once the whole pipeline is going on. As you pointed out in the other PR we are going to need a performance benchmark anyway.